pkg/storage/tpr/storage_interface_test.go

```diff
@@ -126,3 +270,64 @@ func getTPRStorageIFace(t *testing.T, keyer Keyer, restCl restclient.Interface)
 		singularKind: ServiceBrokerKind,
 	}
 }
+
+func serviceCatalogAPIGroup() testapi.TestGroup {
+	// OOPS: didn't register the right group version
+	groupVersion := schema.GroupVersion{Group: servicecatalog.GroupName, Version: "v1alpha1"}
+
+	externalGroupVersion := schema.GroupVersion{
+		Group:   groupName,
+		Version: api.Registry.GroupOrDie(servicecatalog.GroupName).GroupVersion.Version,
+	}
+
+	return testapi.NewTestGroup(
+		groupVersion,
+		servicecatalog.SchemeGroupVersion,
+		api.Scheme.KnownTypes(servicecatalog.SchemeGroupVersion),
+		api.Scheme.KnownTypes(externalGroupVersion),
+	)
+}
+
+func init() {
+	log.SetFlags(log.Lshortfile)
+	testapi.Groups[servicecatalog.GroupName] = serviceCatalogAPIGroup()
+	for k, v := range testapi.Groups {
+		fmt.Printf("----> %s: %s/%s\n", k, v.GroupVersion().Group, v.GroupVersion().Version)
+	}
+}
+
+func verifyStorageError(t *testing.T, err error, errorCode int) {
+	if err == nil {
+		t.Fatal("expected an error, but did not receive one")
+	}
+	storageErr, ok := err.(*storage.StorageError)
+	if !ok {
+		t.Fatalf(
+			"expected a storage.StorageError, but got a %s",
+			reflect.TypeOf(err),
+		)
+	}
+	if storageErr.Code != errorCode {
+		t.Fatalf(
+			"expected error code %d, but got %d",
+			errorCode,
+			storageErr.Code,
+		)
+	}
```

Comment:
can you make this function return an `error`, so that tests that use it don't all fail on the same line?